src/main/java/net/pms/network/SearchRequestHandler.java

```diff
@@ -53,20 +54,23 @@ private DbidMediaType getRequestType(String searchCriteria) {
 		if (matcher.find()) {
 			String propertyValue = matcher.group("val");
 			if (propertyValue != null) {
-				if (propertyValue.toLowerCase().startsWith("object.item.audioitem")) {
-					return DbidMediaType.TYPE_FILES;
-				} else if (propertyValue.toLowerCase().startsWith("object.container.person")) {
+				propertyValue = propertyValue.toLowerCase();
+				if (propertyValue.startsWith("object.item.audioitem")) {
+					return DbidMediaType.TYPE_AUDIO;
+				} else if (propertyValue.startsWith("object.item.videoitem")) {
+					return DbidMediaType.TYPE_VIDEO;
+				} else if (propertyValue.startsWith("object.item.imageitem")) {
+					return DbidMediaType.TYPE_IMAGE;
+				} else if (propertyValue.startsWith("object.container.person")) {
 					return DbidMediaType.TYPE_PERSON;
-				} else if (propertyValue.toLowerCase().startsWith("object.container.album")) {
+				} else if (propertyValue.startsWith("object.container.album")) {
 					return DbidMediaType.TYPE_ALBUM;
-				} else if (propertyValue.toLowerCase().startsWith("object.container.playlistcontainer")) {
```

Comment:
Why remove `toLowerCase()`? Couldn't this cause new problems when renderers ask for `playlistContainer`?

Reply:
I think valib did the "toLowerCase" once before the block.

Reply:
@ik666 you're right, I'm blind :)

Reply:
@SubJunk I also had to look twice :)